Templates/Simple Themes/Modern-Minimal/index.html

```diff
@@ -13,6 +13,11 @@
   href="https://fonts.googleapis.com/css?family=Raleway:700&display=swap"
   rel="stylesheet"
 />
+<link  
+  rel="stylesheet" 
+  href="style.css" 
+  type="text/css"
+>
```

Comment:
looks like this may be missing a closing tag '/'

Reply:
I also thought I was missing it after seeing it was there for the other link tags. I confirmed it works both ways, but W3Schools says no '/' https://www.w3schools.com/html/html_css.asp